src/connector.js

```diff
@@ -0,0 +1,769 @@
+			<p><time datetime="%[short-date-en]" class="text-muted">%[long-date-en]</time> - %[highlightedExcerpt]</p>`;
+	}
+	else {
+		resultTemplateHTML = 
+			`<h3><a class="result-link" href="%[result.clickUri]" data-dtm-srchlnknm="%[index]">%[result.title]</a></h3> 
+			<ul class="context-labels"><li>%[result.raw.author]</li></ul> 
+			<ol class="location"><li>%[result.printableUri]</li></ol> 
+			<p><time datetime="%[short-date-fr]" class="text-muted">%[long-date-fr]</time> - %[highlightedExcerpt]</p>`;
+	}
+}
+
+if ( !noResultTemplateHTML ) {
+	if ( lang === "fr" ) {
+		noResultTemplateHTML = 
+			`<section class="mrgn-tp-lg alert alert-warning">
+				<h2 tabindex="-1" id="wb-land" class="h4">Aucun résultat</h2>
+				<p>Aucun résultat ne correspond à vos critères de recherche.</p>
+				<p>Suggestions:</p>
+				<ul>
+					<li>Assurez-vous que tous vos termes de recherches sont bien orthographiés </li>
+					<li>Utilisez de différents termes de recherche </li>
+					<li>Utilisez des termes de recherche plus généraux </li>
+					<li>Consultez les&nbsp;<a href="/fr/sr/tr.html"> trucs de recherche </a></li>
+					<li>Essayez la <a href="/fr/chaires-recherche/rechercher/recherche-avancee.html">recherche avancée</a></li>
+				</ul>
+			</section>`;
+	}
+	else {
+		noResultTemplateHTML = 
+			`<section class="mrgn-tp-lg alert alert-warning">
+				<h2 tabindex="-1" id="wb-land" class="h4">No results</h2>
+				<p>No pages were found that match your search terms.</p>
+				<p>Suggestions:</p>
+
+				<ul>
+					<li>Make sure all search terms are spelled correctly</li>
+					<li>Try different search terms</li>
+					<li>Try more general search terms</li>
+					<li>Consult the&nbsp;<a href="/en/sr/st.html">search tips</a></li>
+					<li>Try the&nbsp;<a href="/en/sr/srb/sra.html">advanced search</a></li>
+				</ul>
+			</section>`;
+	}
+}
+
+if ( !resultErrorTemplateHTML ) {
+	if ( lang === "fr" ) {
+		resultErrorTemplateHTML = 
+			`<section class="mrgn-tp-lg alert alert-warning">
+				<h2 tabindex="-1" id="wb-land" class="h4">The Canada.ca Search is currently experiencing issues</h2>
+				<p>A resolution for the restoration is presently being worked.	We apologize for any inconvenience.</p>
+			</section>`;
+	}
+	else {
+		resultErrorTemplateHTML = 
+			`<section class="mrgn-tp-lg alert alert-warning">
+				<h2 tabindex="-1" id="wb-land" class="h4">The Canada.ca Search is currently experiencing issues</h2>
+				<p>A resolution for the restoration is presently being worked.	We apologize for any inconvenience.</p>
+			</section>`;
+	}
+}
+
+if ( !querySummaryTemplateHTML ) {
+	if ( lang === "fr" ) {
+		querySummaryTemplateHTML = 
+			`<h2 tabindex="-1" id="wb-land" role="status" class="h4">%[numberOfResults] résultats de recherche pour "%[query]"</h2>`;
+	}
+	else {
+		querySummaryTemplateHTML = 
+			`<h2 tabindex="-1" id="wb-land" role="status" class="h4">%[numberOfResults] search results for "%[query]"</h2>`;
```

Comment:
Instead of repeating the id again and again, I recommend to apply it by default into your core template and try to keep those inner template specific to their use case.

I am not sure that a status can also be totally an heading, I suggest to add some stable heading word with it that way it will be more predictable for the user to find that section.

```suggestion
			`<h2>Résultats de recherche - <span role="status">%[numberOfResults] search results for "%[query]"</span></h2>`;
```